civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -5338,4 +5338,4 @@ function _getManagerContacts($contact_id) {
     }
 
     return $assigned_manager_contact_ids;
-}
+}
```

Comment:
Unnecessary change. Also make sure to have your IDE set so that on file save it makes sure there is a new line at the end of the file